e2e/portfolio-projects/test_portfolio_projects_crud.py

```
@@ -0,0 +1,360 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)